### ISSUE TITLE
What is the status of this project?

### ISSUE THREAD

**Issue by pkramme** [toorop/tmail#27]
Does this project implement a complete SMTP server? Is it still under development?

**Comment by toorop** [comment]
yes it implements a complete SMTP server and as it implements it, it is not in *active* developement.
